Source/Serilog.Exceptions/Filters/CompositeExceptionPropertyFilter.cs

```diff
@@ -0,0 +1,57 @@
+namespace Serilog.Exceptions.Filters
+{
+    using System;
+
+    /// <summary>
+    /// Abstraction over collection of filters that filters property is
+    /// any of given filters alone would filter it. This is equivalent to
+    /// OR over a set of booleans. Executes filters in the order they were
+    /// passed to a constructor.
+    /// </summary>
+    public class CompositeExceptionPropertyFilter : IExceptionPropertyFilter
+    {
+        private readonly IExceptionPropertyFilter[] filters;
+
+        public CompositeExceptionPropertyFilter(params IExceptionPropertyFilter[] filters)
+        {
+            if (filters == null)
+            {
+                throw new ArgumentNullException(
+                    nameof(filters),
+                    "Cannot create composite exception properties filter, because null collection of filters was given");
+            }
+
+            if (filters.Length == 0)
+            {
+                throw new ArgumentException(
+                    "Cannot create composite exception properties filter, because empty collection of filters was given",
+                    nameof(filters));
+            }
+
+            for (int i = 0; i < filters.Length; i++)
+            {
+                if (filters[i] == null)
+                {
+                    throw new ArgumentException(
+                        $"Cannot create composite exception properties filter, because filter at index {i} is null",
+                        nameof(filters));
+                }
+            }
+
+            this.filters = filters;
+        }
+
+        public bool ShouldPropertyBeFiltered(Exception exception, string propertyName, object value)
```

Comment:
Npt sure why my comment is disappearing. How about `CanAddProperty`.

Reply:
Regarding disappearing comments: they are present, but they are collapsed. You need to click 'Show outdated' to view them. 

I will repeat my previous response:
  Could you elaborate a little bit? I am not saying that is a bad name, it's just my gut feeling that 
  is better to phrase the method so that it specifies properties that should be excluded(like 
  ShouldBeFiltered or ShouldBeRejected) then for it to specify properties to be added(like 
  CanAddProperty or ShouldBeAdded).

Reply:
Fair enough.